spot_rl_experiments/spot_rl/envs/gaze_env.py

```diff
@@ -61,30 +59,18 @@ def construct_config_for_gaze(file_path=None, opts=[]):
     if file_path is None:
         config = construct_config(opts=opts)
     else:
-        config = construct_config(file_path=file_path, opts=[])
+        config = construct_config(file_path=file_path, opts=opts)
 
     # Don't need head cameras for Gaze
     config.USE_HEAD_CAMERA = False
-    return config
-
-
-def update_config_for_multiple_gaze(config, dont_pick_up, max_episode_steps):
-    """
-    Updates the config for multiple gaze
-
-    Args:
-        config (Config): Config object
-        dont_pick_up (bool): Whether to pick up the object or not
-        max_episode_steps (int): Max episode steps
 
-    Returns:
-        config (Config): Updated config object
-    """
     # Update the config based on the input argument
-    config.DONT_PICK_UP = dont_pick_up
+    if dont_pick_up != config.DONT_PICK_UP:
+        config.DONT_PICK_UP = dont_pick_up
 
     # Update max episode steps based on the input argument
-    config.MAX_EPISODE_STEPS = max_episode_steps
+    if max_episode_steps is not None:
+        config.MAX_EPISODE_STEPS = max_episode_steps
```

Comment:
add a print here maybe warning that config.MAX_EPISODE_STEPS is being overwritten based on user input

Reply:
Thanks, that is a good suggestion. Added.